Libraries/LibGfx/Bitmap.cpp

```diff
@@ -241,7 +241,7 @@ Bitmap::~Bitmap()
 void Bitmap::strip_alpha_channel()
 {
     VERIFY(m_format == BitmapFormat::BGRA8888 || m_format == BitmapFormat::BGRx8888);
-    for (ARGB32& pixel : *this)
+    for (BGRA8888& pixel : *this)
         pixel = 0xff000000 | (pixel & 0xffffff);
```

Comment:
Wouldn't BGRA imply the B channel in the highest (MS) 8 bits?

Reply:
This is due to the way the type name is now are based on the bytes in memory, but we are accessing it as u32 on a little-endian system, thereby inverting the component order. I went into some more details about this in the commit description I added. Does that make sense?